webview/src/plots/components/ZoomablePlot.tsx

```diff
@@ -45,14 +60,48 @@ export const ZoomablePlot: React.FC<ZoomablePlotProps> = ({
     }
   }, [newSpecsAndData, id, dispatch])
 
-  const handleOnClick = () => dispatch(setZoomedInPlot({ id, plot: plotProps }))
+  useEffect(() => {
+    return () => {
+      window.clearTimeout(enableClickTimeout.current)
+    }
+  }, [])
 
+  const handleOnClick = () =>
+    !clickDisabled.current && dispatch(setZoomedInPlot({ id, plot: plotProps }))
+
+  const commonResizerProps = {
+    onGrab: () => {
```

Comment:
[C] Once I've grabbed the resizer there is no way to bail out, I have to resize the plots before returning them to their original size. Could add the ability to escape as a follow-up?

Reply:
Done!